.circleci/config.yml

```diff
@@ -998,6 +1002,18 @@ jobs:
             circleci step halt
             exit 0
           fi
+
+          image_family=<< parameters.image_family >>
+          if [[ "$CIRCLE_BRANCH" != "master" &&  "<< pipeline.trigger_source >>" != "schedule" ]]; then
```

Comment:
We probably also want to run all integration tests for all tags, just to double check when we cherry-pick commits onto an existing tag.

Reply:
Done